controllers/integration/integration_adapter.go

```diff
@@ -245,20 +245,20 @@ func (a *Adapter) getApplicationSnapshotFromPipelineRun(pipelineRun *tektonv1bet
 
 // getIntegrationTestScenariosForApplication loads from the cluster all IntegrationTestScenarios associated with the given Application.
 // If the Application doesn't have any IntegrationTestScenarios or this is not found in the cluster, an error will be returned.
-func (a *Adapter) getIntegrationTestScenariosForApplication(application *hasv1alpha1.Application) (*[]v1alpha1.IntegrationTestScenario, error) {
-	integrationTestScenarios := &v1alpha1.IntegrationTestScenarioList{}
-	opts := []client.ListOption{
-		client.InNamespace(application.Namespace),
-		client.MatchingFields{"spec.application": application.Name},
-	}
+// func (a *Adapter) getIntegrationTestScenariosForApplication(application *hasv1alpha1.Application) (*[]v1alpha1.IntegrationTestScenario, error) {
```

Comment:
nit: feel free to remove this function instead of commenting it out

Reply:
Removed.

Reply:
Thanks!